extension-base/ctpAzureWrapper.js

```diff
@@ -0,0 +1,77 @@
+// Azure
```

Comment:
Nitpicky but I would recommend file naming with dashes. It's more common in Node land and how we do things through most of our JS code.